tests/views_test.py

```diff
@@ -1,5 +1,100 @@
-from src_api.views import helloworld
+from numpy import ndarray, result_type
+from src_api.views import *
+import shutil
+import os
 
+def TestBasics():
+    """
+    Sets the Basic Folder/Files for Testing.
 
-def test_helloworld():
-    assert helloworld() == "Hello world !"
+    Returns:
+        tuple: filenames and folder.
+    """
+    folder_name = os.path.join(app.config["APP_FOLDER"], "tests", "testfiles")
+    file_name = "test_file.wav"
+    file_name_copy = "test_file_copy.wav"
+    return folder_name,file_name,file_name_copy
+
+
+def TestBasicsCopy(shutil):
+    """
+    Creates Copies of the Files from TestBasics() for Testing.
+    
+    Returns:
+        tuple:  filenames and folder.
+    
+    """
+    folder_name, file_name, file_name_copy = TestBasics()
+    shutil.copyfile(os.path.join(folder_name,file_name), os.path.join(folder_name,file_name_copy))
+    return folder_name,file_name_copy
+
+
+def test_predict_from_file():
+    print()
+
+
+def test_backend_save_request_object():
+    """
+    filetype?
+    """
+    print()
+
+
+def test_backend_rawfile_load():
+    print()
+    
+
+def test_backend_file_load():
+    print()
+
+
+def test_get_duration():
+    print()
+
+
+def test_get_sampling_rate():
+    """
+    Checks if the the Returned Sampling Rate of def_sampling_rate is 8000.
+    """
+    
+    folder_name, file_name_copy = TestBasicsCopy(shutil)
+    result = get_sampling_rate(file_name_copy,folder_name)
+    assert result == 8000
+
+
+def test_load_audio_sequence():
+    """
+    Checks if the shape of the Output of def load_audio_sequence is (8000,)
+    """
+    
+    sampling_rate = 8000
+    max_seq_length = 8000
+    
+    folder_name, file_name_copy = TestBasicsCopy(shutil)
+    result = load_audio_sequence(file_name_copy, sampling_rate, max_seq_length, folder_name)
+    assert result.shape == (8000,)
+
+
+def test_play_sound():
+    print()
+
+
+def test_load_ML_model():
+    print()
+
+
+def test_make_prediction():
+    print()
+
+
+def test_backend_file_delete():
```

Comment:
I would suggest to include making a copy of test files within the function.
WI don't think we can always guarantee that this function is always executed last.
Besides it does not have to be on the audio wav files, any created file (txt format, json, csv etc) should work with this delete function.
I would also test several different paths.

Reply:
delete is included in the functions